src/components/ShareResultsModal/LinkField.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from 'react';
+
+import CheckMarkCircleSVG from './CheckMarkCircleSVG';
+
+export default function LinkField({ url }: Props) {
+  const [isCopied, setIsCopied] = useState(false);
+
+  return (
+    <div className='cio-share-results-feature-group'>
+      <div className='cio-share-results-description'>Share by link</div>
+      <div className='cio-share-results-button-group'>
+        <input className='cio-share-results-link-input' value={url} disabled />
+        <button
+          className='cio-share-results-share-button'
+          type='button'
+          onClick={() => {
+            navigator.clipboard.writeText(url);
```

Comment:
I believe this has the potential to throw an error. Can we wrap it in try catch?

Reply:
![CleanShot 2023-10-31 at 11 20 08@2x](https://github.com/Constructor-io/constructorio-ui-quizzes/assets/7874219/1d19ce26-fdf9-4d59-be17-12da31189b7c)

Added an error for copying as well

Reply:
85a550b81a9df26541cac20a5d7d5af28d154ad3